src/java/picard/analysis/CollectRrbsMetrics.java

```diff
@@ -63,26 +63,24 @@
+            "please see the corresponding GATK <a href='https://www.broadinstitute.org/gatk/guide/article?id=6330'><b>Dictionary</b></a> entry. </p>" +
             "" +
-            "Since cytosine methylation is not exclusive for CpG \"hotspots\", the CollectRrbsMetrics tool outputs a summary table " +
-            "indicating the number of CpG and non-CpG cytosines as well as their conversion C -> T (+ strand) or G -> A (- strand) " +
-            "rates. The tool also outputs the numbers of reads having no CpG sites, and the numbers of reads discarded from the " +
-            "analysis due to inadequate size or excessive numbers of mismatches." +
-            "<br /><br />" +
-            "The tool also provides a table containing detailed information on CpG occurrence frequency, CpG conversion frequencies " +
-            "[C -> T (+ strand) or G -> A (- strand)], and the specific locations of the CpG sites in the genome. The conversion " +
-            "frequency helps determines the methylation status of a CpG site." +
-            "<br /><br />" +
-            "Finally, the tool provides graphical representation of four metrics in the form of a \".pdf\" document. These metrics " +
-            "are the bisulfite conversion rate for CpG and non-CpG cytosines, a distribution of the numbers of CpG sites as a " +
-            "function of CpG conversion rate, the distribution of CpG sites by read coverage, and the numbers of reads discarded due " +
-            "to high numbers of mismatches or inadequate read size." +
+            "<p>Briefly, bisulfite reduction converts un-methylated cytosine (C) to uracil (U) bases.  Methylated sites are not converted " +
+            "because they are resistant to bisulfite reduction.  Subsequent to PCR amplification of" +
+            " the reaction products, bisulfite reduction manifests as [C -> T (+ strand) or G -> A (- strand)] base conversions.  Thus, conversion rates" +
+            " can be calculated from the reads as follows: [CR = converted/(converted + unconverted)].</p>  " +
+
+            "<p>The CpG CollectRrbsMetrics tool outputs three files including summary and detail metrics tables as well as a '.pdf' containing " +
+            "four graphs.  These graphs are derived from the summary table and include a comparison of conversion rates for both CpG and non-CpG sites, " +
+            "the distribution of total numbers of CpG sites as a function of the CpG conversion rates, the distribution of CpG sites by the level of read converage (depth), " +
+            "and the numbers of reads discarded resulting from either exceeding the mismatch rate or size (too short).  The detail metrics " +
+            "table includes the coordinates of all of the CpG sites for the experiment as well as the conversion rates observed for each site.</p>" +
+
+            "<p>Since methylated cytosines are protected against Rrbs-mediated conversion, the methylation rate (MR) can be derived as follows:  " +
```

Comment:
I think this should be appended to the paragraph that comes before the description of the outputs.